### DISCUSSION TITLE
Chocolatey don't see packages in shared folder

### DISCUSSION THREAD

**Discussion by roland-5** [chocolatey/choco#2372] [Q&A]
I have a problem with the stability of the internet connection, so I wanted to download various programs and install / update them on several computers with Windows, from a shared resource via Linux.

I use Debian, and wanted to share folder from my computer, and as I didn't use before samba, I used this tutorial https://fabianlee.org/2016/07/21/ubuntu-creating-a-samba-cifs-share-to-quickly-share-files-with-windows/

Windows didn't want to mount shared folder, because of disabled guest access, so I enabled it with https://docs.microsoft.com/en-us/troubleshoot/windows-server/networking/guest-access-in-smb2-is-disabled-by-default

I disabled chocolatey repo and add my local shared folder from Debian

    choco source add -n="my_server" -s "'//192.168.1.38/shared'" -u=test1 -p=mypassword

Even that, if I search for something I put it there, choco can't find anything. If I mount this shared folder, I can see these files in Windows there.
```
PS C:\Windows\system32> choco source add -n="my_server" -s="\\192.168.1.38\shared\" -u=test1 -p=mypassword
Chocolatey v0.11.1
Updated my_server - \\192.168.1.38\shared\ (Priority 0)
PS C:\Windows\system32> choco source list
Chocolatey v0.11.1
chocolatey [Disabled] - https://community.chocolatey.org/api/v2/ | Priority 0|Bypass Proxy - False|Self-Service - False|Admin Only - False.
my_server - \\192.168.1.38\shared\ (Authenticated)| Priority 0|Bypass Proxy - False|Self-Service - False|Admin Only - False.
PS C:\Windows\system32> choco search r.studio
Chocolatey v0.11.1
0 packages found.
PS C:\Windows\system32> choco install r.studio
Chocolatey v0.11.1
Installing the following packages:
r.studio
By installing, you accept licenses for the packages.
r.studio not installed. The package was not found with the source(s) listed.
 Source(s): '\\192.168.1.38\shared\'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - r.studio - r.studio not installed. The package was not found with the source(s) listed.
 Source(s): '\\192.168.1.38\shared\'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.
```

https://imgur.com/zN3WxcT

I tried push command from Windows like here but it didn't work.

**Comment by TheCakeIsNaOH** [comment]
Can you mount the folder, and try to install from the mounted folder?

**Reply by roland-5** [reply]
Yes, I can execute installer from this shared folder, if I mount it in Windows.

**Reply by TheCakeIsNaOH** [reply]
I mean trying to install the Chocolatey package from the mounted shared folder. Not executing the installer directly from there.
